src/routes/Player/Player.js

```diff
@@ -418,6 +426,7 @@ const Player = ({ urlParams, queryParams }) => {
                 case 'Escape': {
                     closeSubtitlesMenu();
                     closeInfoMenu();
+                    closeSpeedMenu();
```

Comment:
add `closeSpeedMenu()` to the KeyI and KeyS handler.
You could also add Key_ handler for speed, but idk which key to use since KeyS is already taken.
Perhaps we should not add it as its not that common

Reply:
Use KeyR for toggle speed menu. R stands for `rate`